core/platform/models.py

```diff
@@ -281,7 +281,7 @@ def import_bulk_email_services(cls):
             Exception. The value of feconf.BULK_EMAIL_SERVICE_PROVIDER does not
                 correspond to a valid email_services module.
         """
-        if constants.EMULATOR_MODE:
+        if constants.EMULATOR_MODE or not feconf.CAN_SEND_EMAILS:
```

Comment:
I don't think this is correct. Why are we adding this guard? If CAN_SEND_EMAILS is not set in production that does not mean we should import the dev mode services.

Reply:
Rather, if CAN_SEND_EMAILS is not set then we shouldn't even be calling into send_email_to_recipients() in the first place.

Reply:
This is not the call to `send_email_to_recipients` since we don't use mailchimp to send mails. It was the call to the function to update mailchimp database. Modified the user_services function to not call mailchimp functions if emails cannot be sent.